### DISCUSSION TITLE
Should the talk button when viewing the doorbell feed work?

### DISCUSSION THREAD

**Discussion by litex2x** [potmat/homebridge-google-nest-sdm#118] [Q&A]
The talk button is now showing up in the latest iOS update for most of my video feeds. Should it be working for the Nest Doorbell with this Homebridge plugin?

**Comment by potmat** [comment]
Hi @litex2x, unfortunately the SDM API does not support two way audio, so the talk button will not work.  I'm not sure if there's some way to prevent it from showing.
